packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -30,26 +30,93 @@ export interface FluidAppInsightsLoggerConfig {
 		 */
 		mode: "inclusive" | "exclusive";
 		/**
-		 * Controls the default filtering of log events by their category.
+		 * Controls the filtering of log events.
 		 * Leaving this undefined will be treated as an empty array.
```

Comment:
Nit: I would put this part of the comment in a `@remarks` block, since it describes usage notes, rather than a semantic description.

Reply:
Updated.